mopidy_listenbrainz/frontend.py

```diff
@@ -24,6 +26,8 @@ def __init__(self, config, core):
         self.playlists_update_timer = None
 
     def on_start(self):
+        musicbrainzngs.set_useragent('mopidy', mopidy.__version__, 'info@mopidy.org')
```

Comment:
We probably don't want to set this to the official mopidy project email/user agent, since this is a third-party extension.  My email is fine to use for this.

```suggestion
        musicbrainzngs.set_useragent('mopidy-listenbrainz', __version__, 'm@mcarr.one')
```

Reply:
I'm modifying this to get the required information directly from the package metadata.